modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -38,7 +38,7 @@
 #include "core/reference.h"
 #include "core/variant_parser.h"
 
-const char *VisualScriptBuiltinFunc::func_name[VisualScriptBuiltinFunc::FUNC_MAX] = {
+const char *VisualScriptBuiltinFunc::func_name[VisualScriptBuiltinFunc::TEXT_ORD] = {
```

Comment:
Wrong change! `TEXT_ORD` should still be before `FUNC_MAX`. Also, enum order should match name order, and potentially switch order.

Reply:
I apologize. I misinterpreted what you wrote. I will push changes as soon as possible!